src/registrar/models/domain_request.py

```diff
@@ -681,7 +683,7 @@ def delete_and_clean_up_domain(self, called_from):
             logger.error(f"Can't query an approved domain while attempting {called_from}")
 
     def _send_status_update_email(
-        self, new_status, email_template, email_template_subject, send_email=True, bcc_address="", wrap_email=False
+        self, new_status, email_template, email_template_subject, bcc_address="", context=None, **kwargs
```

Comment:
Nitpick:
I really think the extra layers of abstraction make your code harder to follow. I'd get rid of kwargs and just list the args